### ISSUE TITLE
Person popup reading

### ISSUE THREAD

**Issue by PurpurTentakel97** [TechStreamConference/website-frontend#249]
NVDA reads Name, Subtitle, links and than text.
It should be Name, Subtitle, text and then links.
